sdk/storage/azure_storage_blob/src/generated/clients/blob_container_client.rs

```diff
@@ -836,55 +836,53 @@ impl BlobContainerClient {
-                let pipeline = pipeline.clone();
-                async move {
-                    let rsp = pipeline
-                        .send(
-                            &ctx,
-                            &mut request,
-                            Some(PipelineSendOptions {
-                                check_success: CheckSuccessOptions {
-                                    success_codes: &[200],
-                                },
-                                ..Default::default()
-                            }),
-                        )
-                        .await?;
-                    let (status, headers, body) = rsp.deconstruct();
-                    let res: ListBlobsFlatSegmentResponse = xml::from_xml(&body)?;
-                    let rsp = RawResponse::from_bytes(status, headers, body).into();
-                    Ok(match res.next_marker {
-                        Some(next_marker) if !next_marker.is_empty() => PagerResult::More {
-                            response: rsp,
-                            continuation: next_marker,
-                        },
-                        _ => PagerResult::Done { response: rsp },
-                    })
-                }
-            },
-        ))
+                url.query_pairs_mut().append_pair("marker", &marker);
+            }
+            let mut request = Request::new(url, Method::Get);
+            request.insert_header("accept", "application/xml");
+            request.insert_header("content-type", "application/xml");
+            if let Some(client_request_id) = &options.client_request_id {
+                request.insert_header("x-ms-client-request-id", client_request_id);
+            }
+            request.insert_header("x-ms-version", &version);
+            let ctx = options.method_options.context.clone();
+            let pipeline = pipeline.clone();
+            async move {
+                let rsp = pipeline
+                    .send(
+                        &ctx,
+                        &mut request,
+                        Some(PipelineSendOptions {
+                            check_success: CheckSuccessOptions {
+                                success_codes: &[200],
+                            },
+                            ..Default::default()
+                        }),
+                    )
+                    .await?;
+                let (status, headers, body) = rsp.deconstruct();
+                let res: ListBlobsFlatSegmentResponse = xml::from_xml(&body)?;
+                let rsp = RawResponse::from_bytes(status, headers, body).into();
+                Ok(match res.next_marker {
+                    Some(next_marker) if !next_marker.is_empty() => PagerResult::More {
+                        response: rsp,
+                        continuation: next_marker,
+                    },
+                    _ => PagerResult::Done { response: rsp },
+                })
+            }
+        }))
     }
```

Comment:
Example diff of what changed in generated functions for `PageIterator` -> `Pager`.